docs/serving/autoscaling/target-burst-capacity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Target burst capacity"
+linkTitle: "Target burst capacity"
+weight: 50
+type: "docs"
+---
+
+_Target burst capacity_ is a [global and per-revision](./global-vs-per-revision.md) integer setting.
+
+This value specifies how large of a _burst_ of traffic a Knative application can handle without buffering.
+
+If a burst of traffic is too large for the application to handle, the Knative _Activator_ will be placed in the request path to protect the application revision and optimize request load balancing.
```

Comment:
```suggestion
If a desired burst of traffic is too large for the application to handle at current scale, the Knative _Activator_ will be placed in the request path to protect the application revision and optimize request load balancing.
```

Reply:
It's actually in the Google developer docs style guide that we shouldn't use "desire", and I think this would sort of just add fluff anyway.